addons/project/models/project.py

```diff
@@ -749,6 +749,12 @@ def _mail_get_message_subtypes(self):
     def action_project_task_burndown_chart_report(self):
         action = self.env['ir.actions.act_window']._for_xml_id('project.action_project_task_burndown_chart_report')
         action['display_name'] = _("%(name)s's Burndown Chart", name=self.name)
+        context = action['context'].replace('active_id', str(self.id))
+        context = ast.literal_eval(context)
+        context.update({
+            'stage_seq': dict([(stage.name, stage.sequence) for stage in self.type_ids])
```

Comment:
```suggestion
            'stage_seq': {stage.name: stage.sequence for stage in self.type_ids}
```
Isn't that the same?